### ISSUE TITLE
Different learning rates for text encoders and unet in dreambooth setting

### ISSUE THREAD

**Issue by shyammarjit** [huggingface/diffusers#5271]
**Is your feature request related to a problem? Please describe.**

> It has been observed/noted in several studies that text encoders necessitate a diminished learning rate compared to UNet. Please refer to https://rentry.org/59xed3#text-encoder-learning-rate. In order to enhance user adaptability in adjusting distinct learning rates, there is an upcoming proposal to implement disparate rates for UNet and the text encoder.

**Describe the solution you'd like**

* Add the following code lines within the ```parse_args``` function in both ```train_lora_dreambooth_sdxl.py``` and ```train_lora_dreambooth.py``` scripts.
    ```python
    parser.add_argument(
        "--learning_rate_text",
        type=float,
        default=5e-4,
        help="Initial learning rate (after the potential warmup period) to use.",
    )
    ```
* For SDXL, add these lines in ```train_lora_dreambooth_sdxl.py```  replacing https://github.com/huggingface/diffusers/blob/2457599114b8dea7455745fe8b032bbf784b974b/examples/dreambooth/train_dreambooth_lora_sdxl.py#L885C5-L889C6 by 
    ```python
    text_lr = (
        args.learning_rate
        if args.learning_rate_text is None
        else args.learning_rate_text
    )
    
    params_to_optimize = (
        [
            {
                "params": itertools.chain(unet_lora_parameters), 
                "lr": args.learning_rate
            },
            {
                "params": itertools.chain(text_lora_parameters_one),
                "lr": text_lr,
            },
            {
                "params": itertools.chain(text_lora_parameters_two),
                "lr": text_lr,
            },
            
        ]
        if args.train_text_encoder
        else itertools.chain(unet_lora_parameters)
    )
    ```

* For SD model, add these lines in  ```train_lora_dreambooth.py``` replacing https://github.com/huggingface/diffusers/blob/2457599114b8dea7455745fe8b032bbf784b974b/examples/dreambooth/train_dreambooth_lora.py#L967C5-L971C6 by 
    ```python
    text_lr = (
        args.learning_rate
        if args.learning_rate_text is None
        else args.learning_rate_text
    )
    
    params_to_optimize = (
        [
            {
                "params": itertools.chain(unet_lora_parameters), 
                "lr": args.learning_rate
            },
            {
                "params": itertools.chain(text_lora_parameters),
                "lr": text_lr,
            },
        ]
        if args.train_text_encoder
        else itertools.chain(unet_lora_parameters)
    )
    ```

**Describe alternatives you've considered**

**Additional context**

> Please refer to https://rentry.org/59xed3#text-encoder-learning-rate.

**Comment by patrickvonplaten** [comment]
Added here: https://github.com/huggingface/diffusers/pull/5508
